package.json

```diff
@@ -9,6 +9,7 @@
     "@testing-library/user-event": "^12.1.10",
     "axios": "^0.21.0",
     "react": "^17.0.1",
+    "react-axios": "^2.0.3",
```

Comment:
I think we already have the axios dependency, under "axios" I don't know if "react-axios" has some additional functionality that we also need.